cli/command_snapshot_create.go

```diff
@@ -117,9 +120,22 @@ func snapshotSingleSource(ctx context.Context, rep *repo.Repository, u *snapshot
 		return errors.Wrap(err, "cannot save manifest")
 	}
 
-	printStderr("uploaded snapshot %v (root %v) in %v\n", snapID, manifest.RootObjectID(), time.Since(t0))
+	if _, err = policy.ApplyRetentionPolicy(ctx, rep, sourceInfo, true); err != nil {
+		return errors.Wrap(err, "unable to apply retention policy")
+	}
+
+	if ferr := rep.Flush(ctx); ferr != nil {
+		return errors.Wrap(ferr, "flush error")
+	}
+
+	progress.Finish()
+
+	var maybePartial string
+	if manifest.IncompleteReason != "" {
+		maybePartial = " partial"
+	}
 
-	_, err = policy.ApplyRetentionPolicy(ctx, rep, sourceInfo, true)
+	printStderr("\nCreated%v snapshot with root %v and ID %v in %v\n", maybePartial, manifest.RootObjectID(), snapID, time.Since(t0).Truncate(time.Second))
```

Comment:
It turns out that we were parsing this. This modification changed the format and the order.

Filed #272 to track defining the output format for various commands